irc_client/gui.py

```diff
@@ -8,13 +8,48 @@
 import os
 import queue
 import re
+import subprocess
+import sys
 import time
 import tkinter
+import traceback
 from tkinter import ttk
 
 from . import backend, colors, commands
 
 
+def _show_popup(title: str, text: str) -> None:
+    try:
+        if sys.platform == "win32":
+            # FIXME
+            print("Sorry, no popups on windows yet :(")
+        elif sys.platform == "darwin":
+            # https://stackoverflow.com/a/41318195
+            # TODO: can this be indented too?
+            command = (
+                'on run argv\n'
+                '  display notification (item 2 of argv) with title (item 1 of argv)\n'
+                'end run\n'
+            )
+            subprocess.call(['osascript', '-e', command, title, text])
+        else:
+            subprocess.call(["notify-send", f"[{title}] {text}"])
+    except (OSError, subprocess.CalledProcessError):
```

Comment:
`CalledProcessError` only happens with `check_call()`

```suggestion
    except OSError:
```